griptape/utils/observability/griptape_cloud_observability.py

```diff
@@ -0,0 +1,59 @@
+import atexit
+import os
+
+from griptape.utils.observability.griptape_cloud_span_exporter import GriptapeCloudSpanExporter
+from griptape.utils.observability.griptape_instrumentor import GriptapeInstrumentor
+from opentelemetry import trace
+from opentelemetry.trace import Status, StatusCode
+from opentelemetry.instrumentation.threading import ThreadingInstrumentor
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
+
+
+class GriptapeCloudObservability:
```

Comment:
Nit: `Observability` isn't really a noun like the other abstractions in the framework.

Reply:
Technically it is a noun ([dictionary](https://www.merriam-webster.com/dictionary/observable#:~:text=%3A%20capable%20of%20being%20observed%20%3A%20discernible,noun)), but I'd also understand if you just generally dislike the naming choice. Do you have an alternative you'd prefer?